Formula/iconsur.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+class Iconsur < Formula
+  desc "macOS Big Sur Adaptive Icon Generator"
+  homepage "https://github.com/rikumi/iconsur"
+  url "https://registry.npmjs.org/iconsur/-/iconsur-1.6.2.tgz"
+  sha256 "86d0f47221ec638c7157e0857f9e4648f9fee89de443b94c252bd7661747bf35"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"iconsur", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Need to enter the account password for the system user
Is it possible?